app/harvesters/loc_harvester.rb

```diff
@@ -0,0 +1,196 @@
+  #   specifying sets of records to be harvested.
+  #
+  # Initial collections uris:
+  #   https://www.loc.gov/collections/american-revolutionary-war-maps/ (1,251 items)
+  #   https://www.loc.gov/collections/civil-war-maps/ (1,707 items)
+  #   https://www.loc.gov/collections/panoramic-maps/ (1,433 items)
+  #
+  # @see .expected_opts
+  def initialize(opts = {})
+    opts[:uri] = '' if opts[:api] # don't require a URI if we have api opts
+
+    opts[:name] ||= 'loc'.freeze
+    opts[:api]  ||= {}
+
+    opts[:api][:uris]          = Array.wrap(opts[:api][:uris])
+    opts[:api][:item_base_uri] = 'https://www.loc.gov/item/'.freeze
+
+    @http = Krikri::AsyncUriGetter.new(opts: {follow_redirects:  true,
+                                              inline_exceptions: true})
+    super
+  end
+
+  ##
+  # @return [Hash] A hash documenting the allowable options to pass to
+  #   initializers.
+  #
+  # @see Krikri::Harvester::expected_opts
+  def self.expected_opts
+    { 
+      key: :api,
+      opts: {
+        uris:          { type: :string, required: true,  multiple_ok: true},
+        item_base_uri: { type: :string, required: false, multiple_ok: false}
+      }
+    }
+  end
+
+  ##
+  # @param identifier [#to_s] the identifier of the record to get
+  # @return [#to_s] the record
+  def get_record(identifier)
+    build_record(request(build_record_uri(identifier)))
+  end
+
+  ##
+  # @see Harvester#record_ids
+  def record_ids
+    enumerate_uris.map { |u| u.split('/').last }.to_enum
+  end
+
+  private
+
+  ##
+  # @private
+  def build_record_uri(identifier)
+    RDF::URI(opts[:item_base_uri]) / identifier.to_s / JSON_PARAM
+  end
+
+  ##
+  # Overrides ApiHarvester.get_identifier because the idenitifer field
+  # for Library of Congress
+  #
+  # @param doc [Hash] a JSON serialziation with an identifier
+  #
+  # @return [String] the provider's identifier for the document
+  def get_identifier(doc)
+    # doc is the response to a request for /item/<id>
+    doc['item']['library_of_congress_control_number']
+  end
+
+  ##
+  # @param response [Hash] a response from the LoC API
+  #
+  # @return [Integer] value of the number of records in the response
+  def get_count(response)
+    response['pagination']['of']
+  end
+
+  ##
+  # @param response [Hash] a response from the LoC API
+  #
+  # @return [Array] the documents in the response
+  def get_docs(response)
+    response['results']
+  end
+
+  ##
+  # Get the URL for the next request.
+  #
+  # @param opts [Hash] the response to the current request. The next page is
+  # provided in the ['next'] element
+  #
+  # @param response [Hash]
+  #
+  # @return [Hash] the next request's options hash
+  def next_options(response)
+    response['pagination']['next']
+  end
+
+  ##
+  # @return [Enumerator] an enumerator over the records
+  def enumerate_records
+    enumerate_uris.lazy.map do |item_uri|
+      @http.add_request(uri: URI.parse(item_uri + JSON_PARAM))
```

Comment:
I think we have: https://github.com/dpla/heidrun/pull/52/files#diff-1a4317d11c81f5bff6982f05e748c4bfR35

Is there something I'm overlooking?